src/aws-cpp-sdk-core/source/utils/UUID.cpp

```diff
@@ -86,5 +88,39 @@ namespace Aws
 
             return UUID(randomBytes);
         }
+
+        UUID UUID::PseudoRandomUUID()
+        {
+            static size_t randomSeed = []()
+                    {
+                        std::random_device rd;
```

Comment:
still not sure why it needs to be initialized here? its not static so we can pull into constructor?